test-cases/longevity/longevity-parallel-schema-changes-12h.yaml

```diff
@@ -18,6 +18,7 @@ n_loaders: 2
 n_monitor_nodes: 1
 
 instance_type_db: 'i4i.2xlarge'
+instance_type_runner: 'r5.2xlarge'
```

Comment:
As we discussed, I suggest trying with m7i-flex.xlarge or m6i.xlarge if the former isn't available on all regions.

Reply:
changed the default to m7i-flex